app/main/models/models.py

```diff
@@ -123,7 +130,8 @@ def format(self):
             'project_id': self.project_id,
             'child_clusters': [cluster.id for cluster in self.child_clusters],
             'child_items': [item.format() for item in self.child_items],
-            'parent_cluster': self.parent_cluster_id
+            'parent_cluster': self.parent_cluster_id,
+            'total_items': self.count_items(0)
```

Comment:
Another indicator that the count parameter is redundant, you shouldn't need to pass a 0 here.